mitodl/frontend-app-learning
# style: Update texts/design of course upgrade card

#### Related ticket:
https://github.com/mitodl/mitxonline/issues/556

#### What this PR does?
- Updates the style of the course upgrade card as per required in MITxOnline
- Adds configuration values to set the text for the title and description
- Updates the card code to use the text from the environment
- Updates the Upgrade button text to skip showing price

#### How should this PR be tested?
- Setup your edX devstack to use this branch for `frontend-app-learning` (You could simply add this fork's remote and checkout this branch)
- Do a `make frontend-app-learning-shell`, in the shell run npm run build
- Restart your MFE container `docker-compose restart frontend-app-learning`
- To see the upgrade dialog you need to set up a few things mentioned in https://github.com/mitodl/mitxonline/issues/567#issuecomment-1131656687
- Set title/description environment values want to see in the `.env.development` file 
- Once you land on the course details page you should be able to see the updated changes
- Once above is done:
    - You should see the updated card title
    - You should see the updated card descriotion
    - You should see the `Upgrade` text only in the Upgrade button (That used to be `Upgrade for $149` etc)

#### Screenshot 
**Desktop**
<img width="1677" alt="Screenshot 2022-05-31 at 2 49 28 PM" src="https://user-images.githubusercontent.com/34372316/171146049-fd2a9744-9512-4931-82b3-0adb092bb52e.png">

**Mobile**
<img width="498" alt="Screenshot 2022-05-31 at 2 49 39 PM" src="https://user-images.githubusercontent.com/34372316/171146088-fd31becb-5832-419a-a7be-f093f6bc6246.png">
